modules/onboarding/src/test/kotlin/edu/stanford/spezi/module/onboarding/sequential/SequentialOnboardingViewModelTest.kt

```diff
@@ -0,0 +1,67 @@
+package edu.stanford.spezi.module.onboarding.sequential
+
+import com.google.common.truth.Truth.assertThat
+import edu.stanford.spezi.core.navigation.Navigator
+import edu.stanford.spezi.core.testing.CoroutineTestRule
+import edu.stanford.spezi.core.testing.runTestUnconfined
+import io.mockk.coEvery
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class SequentialOnboardingViewModelTest {
+    private val navigator: Navigator = mockk(relaxed = true)
+    private val repository: SequentialOnboardingRepository = mockk(relaxed = true)
+
+    @get:Rule
+    val coroutineTestRule = CoroutineTestRule()
+
+    private lateinit var sequentialOnboardingViewModel: SequentialOnboardingViewModel
+
+    @Before
+    fun setup() = runBlocking {
```

Comment:
runBlocking is not needed as you are already using `coEvery`. Also below `verify { runBlocking` can be replaced with `coVerify {`